types/nightwatch/test/testDescribe.ts

```diff
@@ -0,0 +1,85 @@
+import { ExtendDescribeThis } from "nightwatch";
+
+// TODO: uncomment after fixing async/await issue
+// describe('capture browser exceptions', function () {
+//     it('does', async function () {
+//         await browser.captureBrowserExceptions(event => {
+//             console.log('>>> Exception:', event);
+//         });
+
+//         await browser.navigateTo('https://duckduckgo.com/');
+//         const aboutLink = await browser.findElement('#logo_homepage_link');
+
+//         await browser.executeScript(
+//             function (aboutLink) {
+//                 aboutLink.setAttribute('onclick', 'throw new Error("Hello world!")');
+//             },
+//             [aboutLink],
+//         );
+
+//         await browser.click(aboutLink);
+//     });
+// });
```

Comment:
can you please, uncomment this and update the tests

Reply:
But uncommenting this won't work as the `async/await` issue is not yet fixed in this PR. Should I remove this?

Reply:
ahh! my bad. No, we can leave it there.